src/prefect/client/secrets.py

```diff
@@ -41,6 +41,10 @@
 export PREFECT__CONTEXT__SECRETS__MY_KEY="MY_VALUE"
 ```
 
+::: tip Default secrets
+Special default secret names can be used to authenticate to third-party systems in a installation-wide way. Read more about this in our [Secrets concept documentation](../../../core/secrets.md#default-secrets).
```

Comment:
FYI I recently learned that if you provide a doc path staring with `/` from the root you can avoid getting the directory structure right.  So in this case, `"/core/secrets.md#default-secrets"` should do the trick

Reply:
Ahhh this is so much better (used here in cc15fb5)